test/support/email_helper.rb

```diff
@@ -0,0 +1,30 @@
+module EmailHelper
+
+  # get the last email sent by ActionMailer
+  #
+  # Returns
+  #   last email or nil if none sent
+  def last_email_delivered
+    ActionMailer::Base.deliveries.last
+  end
+
+  # extract a link from an email
+  #
+  # Parameters
+  #   email: the email returned by `last_email_delivered`
+  #
+  # Returns
+  #   string of link if found or nil
+  def extract_link_from(email)
```

Comment:
Maybe change to "extract first link from email"? tiny nitpick and im fine to merge without this though

Reply:
ah fair point. it used to be as you suggest but aesthetically it reads like english: `extract_link_from(email)` as opposed to `extract_link_from_email(email)`. i believe this is supposed to be "self documenting code" but i can make it explicit